api/src/controllers/reviews.ts

```diff
@@ -165,7 +165,17 @@ router.post("/", async function (req, res, next) {
  * Delete a review
  */
 router.delete('/', async (req, res, next) => {
-  if (req.session.passport?.admin) {
+
+  const checkUser = async () => {
+
+    let review = await getDocuments(COLLECTION_NAMES.REVIEWS, {
+      _id: new ObjectID(req.body.id)
+    });
+
+    return review.length > 0 && review[0].userID === req.session.passport?.user.id;
```

Comment:
Just wondering, are we checking that the user IDs match just for extra assurance? 

 In theory (assuming our data isn't being intercepted / tampered with), it shouldn't be possible for the returned reviews to belong to a different user right?

Reply:
Yes, the check is just for assurance that the author of the review is the one who is trying to delete it. It shouldn't be possible for the review to belong to a different user unless the user sends a request to delete a review that doesn't belong to them.